NEWS

```diff
@@ -3,6 +3,18 @@
 History
 -------
 
+1.3.0 (2017-12-XX)
```

Comment:
This looks ready to merge for me.
Just a reminder, this date needs to be updated when doing the release. :)